tests/client-dbus/tests/udev/_utils.py

```diff
@@ -58,9 +58,7 @@ def random_string(length):
     :param length: Length of random part of string
     :return: String
     """
-    return "stratis_{0}".format(
-        "".join(random.choice(string.ascii_uppercase) for _ in range(length))
-    )
+    return "".join(random.choice(string.ascii_uppercase) for _ in range(length))
```

Comment:
Whoops, this one was not unncessary....need the f-string.

Reply:
OK, should be fixed.  (I encountered 2 merge conflicts, and lost one of the 3 commits from before, but it's probably because it was involved on this line.)